### ISSUE TITLE
[Bug|Crash]: werid visual in game with some skin

### ISSUE THREAD

**Issue by Wither-Skeleton** [Yesssssman/epicfightmod#1179]
### Have you checked if a similar issue is already reported by someone else?

- [X] I checked there are no similar issues have been reported.

### Have you read the support policy?

- [X] I read it and I accept the policy.

### Are you using the latest Epic Fight and recommended Forge version?

- [X] I checked I'm using latest Epic Fight and recommended Forge version.

### Is this issue related to mod incompatibility?

- [ ] This is a mod compatibility issue and I'm aware of the problem.

### Minecraft Version

1.18.2

### What happened?

my skin basically have a glass as overlay and it shown like:
![image](https://user-images.githubusercontent.com/43639224/189538123-adb55b05-085f-4fa8-b982-ff55bcf80583.png)

**Comment by MetalKnight56** [comment]
Send a screenshot of you skin on mining mode

**Comment by Wither-Skeleton** [comment]
> Send a screenshot of you skin on mining mode

![image](https://user-images.githubusercontent.com/43639224/189538236-7e7bdafe-f77d-4b92-ac36-033a202e4a5e.png)

**Comment by MetalKnight56** [comment]
What forge version and epic fight version are you on?

**Comment by Wither-Skeleton** [comment]
> What forge version and epic fight version are you on?

epicfight 18.3.4 
forge 40.1.73

**Comment by Woodgoldtwo** [comment]
It seems like Efm's custom model is not compatible with outer layer transparency

**Comment by MetalKnight56** [comment]
> It seems like Efm's custom model is not compatible with outer layer transparency

Yes we are ware of that, we are already working on a fix for it, just wanted to make sure he was on latest and that everything was working fine.

**Comment by MetalKnight56** [comment]
> > What forge version and epic fight version are you on?
> 
> epicfight 18.3.4 forge 40.1.73

Try updating, we've fixed multiple issues regarding transparency in latest releases

**Comment by Yesssssman** [comment]
Fixed in 18.3.6.
